src/mqttcm_connect.c

```diff
@@ -1071,8 +1112,9 @@ rbusError_t MqttSubscribeSetHandler(rbusHandle_t handle, rbusProperty_t prop, rb
 
 		if(type_t == RBUS_STRING) {
 			char* data = rbusValue_ToString(paramValue_t, NULL, 0);
+		//	char *data = "mesh";
 			if(data) {
-				if(((strcmp (data, "Webconfig") == 0)) || (strcmp (data, "Mesh") == 0))
+				if(strcmp (data, "Webconfig") == 0)
```

Comment:
Better to add a macro defined in mqttcm_webcfg.h as "SUBSCRIBE_WEBCONFIG" and use it , modify it in line 1509 also.